src/components/Settings/Advanced/Advanced.tsx

```diff
@@ -74,39 +75,72 @@ const initialSettings: SettingsItem[] = [
     label: 'Factory reset',
     visible: true,
     caseSensitive: false
-  },
-  {
-    key: 'back',
-    label: 'Back',
-    visible: true
   }
 ];
 
 export const AdvancedSettings = () => {
   const dispatch = useAppDispatch();
-  const { data: globalSettings, isSuccess } = useSettings();
+  const { data: globalSettings, isSuccess: isSettingsSuccess } = useSettings();
   const updateSettings = useUpdateSettings();
   const [activeIndex, setActiveIndex] = useState(0);
   const bubbleDisplay = useAppSelector((state) => state.screen.bubbleDisplay);
   const { refetch: fetchDeviceStatus } = useDeviceInfo();
   const { data: rootPW } = useRootPassword();
 
+  const { data: manufacturingSettings, isSuccess: isManufacturingSuccess } =
+    useManufacturingSchema();
+
   const updatedSettings = useMemo(() => {
-    if (!isSuccess) {
+    if (!isSettingsSuccess) {
       return initialSettings.map((item) => ({
         ...item
       }));
     }
-    return initialSettings.map((item) => ({
+    const formattedInitialSettings = initialSettings.map((item) => ({
       ...item,
       label:
         item.key === 'root_password'
-          ? `${item.label}: ${rootPW || 'Loading...'}`
```

Comment:
may I ask why? suggesting it is already there but cannot be displayed is different from loading imho?